laboratory/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
Can you please add a `concurrency` definition similar to https://github.com/WalletConnect/rpc-proxy/blob/master/.github/workflows/ci.yaml#L18-L24